src/cache/cache_las.c

```diff
@@ -755,6 +755,9 @@ __wt_las_insert_block(WT_CURSOR *cursor,
 				    &las_timestamp, upd->prepare_state,
 				    upd->type, &las_value);
 
+			if (upd->prepare_state == WT_PREPARE_INPROGRESS)
```

Comment:
I think we should increment "txn_prepared_updates_in_lookaside" after a successful cursor->update call. If update() fails for some reason, we shouldn't increment this value.

Reply:
Is the idea that WT-4346 will take care of decrementing this counter when prepared updates get removed ?
At the moment this counter doesn't get decremented, which means it is rather a count of updates that have been put in the lookaside till now, and not a count of updates that exist in the lookaside at the moment.

Reply:
Currently there is no statistics for counting the removals from lookaside, as we need to get the value of the record to examine the prepared flag, which will have some performance impact. my opinion is prepared_updates_lookaside_inserts will tell whether prepared updates are added to cache overflow mechanism or not which some what represents that life span of prepared updates. measuring removals will be useful to test our changes correctness but will not represent much user scenario diagnostics.